src/main/java/code401challenges/MultiBracketValidation.java

```diff
@@ -0,0 +1,40 @@
+package code401challenges;
+
+import org.apache.commons.math3.exception.NullArgumentException;
+
+import java.util.Stack;
+
+public class MultiBracketValidation {
+    public static boolean multiBracketValidation(String s) {
+        //set a placeholder for the current string
+        String curChar = null;
+        if (s == null) {
+            throw new NullArgumentException();
+        }
+//       create stacks for
+        Stack<String> lBrackets = new Stack<>();
+
+// ********************* how could i do this recursively?****************************
+        for (int i = 0; i < s.length(); i++) {
+//            take in the current char and convert it to string
+            curChar = String.valueOf(s.charAt(i));
+//            Check if the first bracket is a right side bracket return false if it is.
+            if (curChar.equals("}") && lBrackets.empty() ||
+                    curChar.equals("]") && lBrackets.empty() ||
+                    curChar.equals(")") && lBrackets.empty()) {

```

Comment:
Make this an "else if" statement, put it after the "if" statement involving the opening brackets. Also, you don't want to automatically return false. You want to pop the stack and return false if the bracket on top doesn't match.